cves/kernel/CVE-2013-6383.yml

```diff
@@ -448,7 +460,7 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: Failure to consider that this file was not always properly restricted.
```

Comment:
Expand on this some more. What are good practices that find things that people forgot? How can you find permissions issues? This seems like a lapse, but are there other mistakes involved?

Reply:
Added more details and suggestions listed above.